udfs/community/test_cases.yaml

```diff
@@ -408,6 +408,11 @@ levenshtein:
   - test:
     input: CAST('Connecticut' AS STRING), CAST('CT' AS STRING)
     expected_output: CAST(10 AS INT64)
+pi:
+  - test:
+    input: (NULL)
+    expected_output: CAST(3.141592653589793 AS FLOAT64)
+
```

Comment:
Let's remove unit test for this UDF because all it's really doing is proving that built-in BigQuery function ACOS works as expected. Also the unit testing framework expects input arguments and the pi UDF doesn't have any.

Reply:
@danieldeleo no problem, I have removed it now thanks!